workflow/rocoto/gefs_tasks.py

```diff
@@ -9,48 +10,63 @@ def __init__(self, app_config: AppConfig, cdump: str) -> None:
         super().__init__(app_config, cdump)
 
     def stage_ic(self):
-
         cpl_ic = self._configs['stage_ic']
-
         deps = []
-
+        dtg_prefix = "@Y@m@d.@H0000"
+        if self._configs['base']['END_OF_IAU_START']:
+            offset = "+03:00:00"
+        else:
+            offset = None
```

Comment:
This should be handled better. Hard-wiring offsets deep in this code will make this difficult to maintain.
This is a capability that can also be used in GFS if someone wants to start a forecast from the end of the assimilation window e.g. a reanalysis